gitops/prer/create_gitops_prs.go

```diff
@@ -183,7 +183,7 @@ func main() {
 		for _, target := range targets {
 			log.Println("train", train, "target", target)
 			bin := bazel.TargetToExecutable(target)
-			exec.Mustex("", bin, "--nopush", "--nobazel", "--deployment_root", gitopsdir)
+			exec.Mustex("", bin, "--nopush", "--deployment_root", gitopsdir)
```

Comment:
remind me why we had `--nobazel` originally?

Reply:
--nobazel is not used anywhere currently. It was an early attempt to make sure the binary is working under `bazel run` as well as direct execution of a script in bazel-bin/ as create_gitops_prs run it. Later it was achieved without using of any command line flags but this historical artifact remains.